src/test/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " is working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(getName() + " has finished working.");
```

Comment:
Similarly, when stopping work, the message should clearly indicate that a `Bulldozer` has finished working.